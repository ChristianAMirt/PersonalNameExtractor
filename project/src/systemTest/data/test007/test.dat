<NER> Author names include Dean Koontz and Anne Rice.</NER>