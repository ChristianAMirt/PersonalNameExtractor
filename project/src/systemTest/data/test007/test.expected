<NER> Author names include <PER> Dean Koontz </PER> and <PER> Anne Rice </PER> .</NER> 
